.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run tests
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      buildkit-tag:
+        description: "The isle-buildkit tag to pull for the fleet of docker containers"
+        required: true
+        type: string
+        default: 'main'
+      starter-site-ref:
+        description: "The islandora-starter-site ref to checkout (heads/BRANCH-NAME or tags/TAG-NAME)"
+        required: true
+        type: string
+        default: 'heads/main'
+  schedule:
+    - cron: '15 11 * * *'
```

Comment:
Why test with cron? Do we expect this to fail without changes to the code base?

Reply:
Yeah to catch issues with buildkit+starter that are otherwise never discovered

Reply:
Thought being: this is sort of the last mile before someone installs a site, so if some issue does arise between buildkit and starter site this feels like the best place to discover it since it's the place those two repos converge